scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/activities/ActivityCreationTemplate.java

```diff
@@ -369,7 +370,25 @@ public Duration valueAt(final Duration start) {
           null,
           null,
           true));
-    } else{
+    } else if (this.type.getDurationType() instanceof DurationType.Fixed dt) {
+      //select earliest start time, STN guarantees satisfiability
+      final var earliestStart = solved.start().start;
+
+      // TODO: When scheduling is allowed to create activities with anchors, this constructor should pull from an expanded creation template
+      return Optional.of(SchedulingActivityDirective.of(
+          type,
+          earliestStart,
+          dt.duration(),
+          SchedulingActivityDirective.instantiateArguments(
+              this.arguments,
+              earliestStart,
+              facade.getLatestConstraintSimulationResults(),
+              evaluationEnvironment,
+              type),
+          null,
+          null,
+          true));
+    } else {
      throw new UnsupportedOperationException("Duration type other than Uncontrollable and Controllable are not suppoerted");
```

Comment:
The exception message here can use a little update:
```suggestion
     throw new UnsupportedOperationException("Duration types other than Uncontrollable, Controllable, and Fixed are not supported: " + this.type.getDurationType());
```